pingcap/tidb
# expression: Support Sqrt, Ceil, Floor and CastIntAsReal push down to TiFlash

### What problem does this PR solve?
Related to https://github.com/pingcap/tics/issues/1901
Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?


What's Changed:
Add Floor, Ceil, Sqrt and CastIntAsReal(for pushing down a sqrt for an int)  into scalarExprSupportedByFlash list

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
Use tiflash/integrated/ops or algernon to run a tidb-server and use explain query to make sure corresponding expression is pushed down

Side effects


### Release note 
- Pushdown sqrt, ceil, floor and CastIntAsReal to tiflash
